samples/VSSample.Tests/VSSample.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <!-- We Move to central package management: For version details, please refer to samples/Directory.packages.props -->
```

Comment:
Can we change "We Move to central" to "We use central..."?

Also, can we give instructions about how they can override the versions?